### DISCUSSION TITLE
optimize Developer tools

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#9298] [Configuration]
As a followup of Discord discussion #beta 2021.6 please consider optimizing the Developer tools page/tabs.

On a fully loaded system with many entities, the default /state page takes a long time to load, and become truly responsive for entering filtering strings (even on desktop, which is needed for viewing attributes column, which is not there on mobile.) one even has to wait for that when in need of the `/template` or other tabs

Probably the easiest/quickest fix would be loading the `/template` page by default, instead of the `/state`

Discussion suggested the /state page would be optimized too, but I am no dev so wouldn't know how to start with that invitation...

please consider?

**Comment by BoneheadFraggle** [comment]
Or loading the /state page, but with all entities filtered out at first so you instead need to enter something (for example using the new very nice wildcards)
A simple * would show you everything you see today, and all other times you open the page you would save time :-)

**Reply by emufan** [reply]
This!

**Comment by GrizzlyAK** [comment]
Speaking of the `/template` page, it would be great if the `RESET TO DEMO TEMPLATE` button was replaced with a `CLEAR` button. Or better yet, just _ADD_ a `CLEAR` button to the page. The `CLEAR` button could place the cursor in the newly emptied box ready for a paste or typing.
